build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
My test cases for this was:

```hbs
{{moment}}<br>
{{moment '2012-02-02'}}<br>
{{moment format="YYYY"}}<br>
{{moment '2014-05-06' format="YYYY-MM"}}
```

It was always defined under these circumstances. I believe it defaults to an empty object.